backend/src/main/java/ai/giskard/web/dto/ComparisonType.java

```diff
@@ -0,0 +1,22 @@
+package ai.giskard.web.dto;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Getter
+public enum ComparisonType {
+    IS(true, "==", "giskard.slicing.slice.EqualTo(%s, %s)"),
```

Comment:
this is going to be hard to maintain if the backend becomes aware of python implementation details. I think we should just pass a list of rules (language agnostic) from the front to python and then python would know how to convert for example "IS_NOT" to `NotEqualTo`